### ISSUE TITLE
3.0.1 - keyboard nav is now ......s...l...o...w

### ISSUE THREAD

**Issue by freste** [leggett/simplify#636]
keyboard navigation is very slow since latest update. E.g., w/simplify off, j/k in the msg list moves immediately. With Simplify on, it takes about 1/2 second to move.
---
Simplify v.3.0.1 - Configuration: aAX boldHighlight bundleInbox ca3 caI caL cfC cfO cfQ cfR cfU composeActions dateGroup fontMsg hideListCount hideMsgCount hideTabIcons highDensity inInbox inList invertAddons invertCompose izBgDefault listWidthLg mediumTheme minimizeChat msgWidthSm rightStar simplify supExtWarn - System: Mozilla/5.0 (X11 CrOS aarch64 14150.87.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/94.0.4606.124 Safari/537.36 - Window: 1536 x 864 (100pct zoom) - Language: en

**Comment by leggett** [comment]
It shouldn't be slow. Try a hard refresh (Shift + click on browser refresh button). Let me know if that doesn't fix it.

**Comment by freste** [comment]
Sorry, reload produced no change (I even went as far as rebooting this chromebook); I can send a screen video if you like, but not until tomorrow. 1/2 second might be a bit of an exaggeration, but definitely slow enough to be annoying :-)  
BTW, I found 'debug mode', turned it on; no timing info in the console that I could see. (I can send you those msgs if helpful.)

PS As per #635, no profile pictures displayed. But they suddenly showed up after I turned debug mode on. (Coincidence?) They stuck around when switched that off, reloaded.

**Comment by leggett** [comment]
The profile pics was a separate issue and I pushed v3.0.1 to fix it this morning. 

A screen video would be helpful. You can email it to me at hi@simpl.fyi so it isn't posted here. Also, do you have any other extensions running that have access to Gmail? Maybe there is a conflict?

**Comment by freste** [comment]
Just noticed it's back to being fast again! looks like I'm at 3.0.2 now, hopefully that's expected.
(Note - I haven't tried on the other machine yet that I originally saw the problem on.)

**Comment by leggett** [comment]
Got your video. And ouch, that was painfully slow. I can actually hold down the J key and it feels as fast as Gmail for me. So that is what I'm aiming for.

I need to test more with the inbox tabs turned on.

**Comment by leggett** [comment]
Seems that this issue was solved in 3.0.2. Re-open if it comes back.
